docs/source/notes/cuda.rst

```diff
@@ -75,7 +75,8 @@ matmul and convolutions are controlled separately, and their corresponding flag
   # The flag below controls whether to allow TF32 on matmul. This flag defaults to True.
   torch.backends.cuda.matmul.allow_tf32 = True
 
-  # The allow_tf32 flag for convolutions is not implemented yet
+  # The flag below controls whether to allow TF32 on cuDNN. This flag defaults to True.
```

Comment:
In the preceding paragraphs it should be "matmuls" and "flags"

Reply:
TF32 is also being enabled for RNNs, not just matrix multiplications and convolutions, so these should be mentioned in the preceding paragraphs, too

Reply:
Aside: it seems challenging for users to know if they're using cuDNN, but since we're hoping regular users won't need to think about this flag that's probably OK.

Reply:
fixed. A new paragraph is added after this piece of code.

Reply:
@zasdfgbnm Hello! How does grid sample uses matmuls or convolutions? I could not find any code evidence.

Reply:
This merge request is the wrong place to ask these types of questions. Please use the [discussion board](https://discuss.pytorch.org/) for it.